### ISSUE TITLE
Access BusyTrackerService

### ISSUE THREAD

**Issue by mtpultz** [victos/angular-opensource#19]
Would it be possible to get access to the instance of BusyTrackerService?  I've tried to dependency inject it into my app and parent components so I could tell globally if anything was busy, but even though I've got the import correct it doesn't seem to exist.

**Comment by victos** [comment]
No, there's no way to get the instance of BusyTrackerService now.

**Comment by mtpultz** [comment]
Would it be possible to get a public version that simply provides an indicator that someone is busy, and maybe a key for indicating the name?  It would be useful to have a service like this so adjacent components would know not to allow things like routing when busy is running.
